src/http/client.js

```diff
@@ -30,6 +31,35 @@ export default class HttpClient {
 		}
 	}
 
+	/**
+	 * Perform an http request using the given method
+	 * @param {String} method
+	 * @param {String} url
+	 * @param {Object} headers
+	 * @param {Object} body
+	 * @return {Object}
+	 */
+	async request(method, url, headers, body) {
+		const contentType =
+			body instanceof FormData
```

Comment:
Can't it be a property of form (like file) the way of differenciating between form-data. InstanceOf is weird.

Reply:
how? :(

https://developer.mozilla.org/en-US/docs/Web/API/FormData

Reply:
Yep, it's not feasible, I thought both objects were yours or belonged to the library. One is a string and string checking is horrible, the other is hard to recognize and do a if (body.append) is uglier than the instanceof and more error prone (in that case :P).

Reply:
I feel better now: 
https://github.com/wedeploy/wedeploy-sdk-js/blob/master/src/api/node/NodeTransport.js#L126